pip/index.py

```diff
@@ -1184,3 +1190,13 @@ def is_wheel(self):
 # An object to represent the "link" for the installed version of a requirement.
 # Using Inf as the url makes it sort higher.
 INSTALLED_VERSION = Link(Inf)
+
+
+Search = namedtuple('Search', 'supplied canonical formats')
+"""Capture key aspects of a search.
+
+:attribute user: The user supplied package.
```

Comment:
Its fixed higher up,and travis is being slow, so I'll ignore this for now unless we need to respin for some other reason